vuepress/docs/next/docs/compose/local-hub-overview.md

```diff
@@ -1,14 +1,14 @@
 ---
 sidebarDepth: 2
 ---
-# Entando Component Repository
+# Local Hub
 
 ## Introduction
 
-The Entando Component Repository (ECR) is a repository where 
+The Local Hub is a repository where 
```

Comment:
Need to reword this sentence. You can no longer share components from an ECR/Local Hub to other apps or enterprises.